### ISSUE TITLE
Recombinant lineage nomenclature miscBA1BA2PostX

### ISSUE THREAD

**Issue by ktmeaton** [yatisht/usher#235]
This week (2022-04-17), I noticed the [public trees](http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/) have a new(?) nomenclature for recombinant lineages that takes the format `miscBA1BA2PostX`. Some specific examples:

- `miscBA1BA2Post8393` | EPI_ISL_11360224
- `miscBA1BA2Post17k` | EPI_ISL_11367575

Could you provide some guidance on what this nomenclature means? Thanks!

**Comment by ktmeaton** [comment]
That makes perfect sense thank you! Giving these clades names is super helpful, otherwise they previously got misassigned to `XM` just as you indicate. @AngieHinrichs can I ask how you do the breakpoint detection? I'm guessing `ripples` in this application?

**Comment by corneliusroemer** [comment]
My guess would be Angie uses the breakpoint of the pango designated
recombinant - one can look up on the issue where it is.

**Comment by ktmeaton** [comment]
Wow, you preemptively answered all of my follow-up questions!

> At this point, the misc/proposed labels are replaced with BA.2 or BA.1 depending on which parent lineage donated more mutations to the potential recombinant.

I was wondering if we would start to see the "misc" labels in pangolin, but understand it's better to stick to the designated lineages!

**Comment by ilevade** [comment]
Hi ! Following Katherine's question and your first answer Cornelius, we actually proposed an issue for the sample "miscBA1BA2Post8393" | EPI_ISL_11360224

https://github.com/cov-lineages/pango-designation/issues/557

But if I understood correctly there were not enough sequences to propose a Pango name in this case. I found other sequences with the same break point (11537-12880), and I added them to the issue. In our case they were assigned XE by NextClade, and not XM.

**Comment by AngieHinrichs** [comment]
Yes @ilevade, that is a nice cluster of sequences, with several mutations that distinguish them from other recombinants with a similar breakpoint.  But so many distinct recombinants were detected that it got a little overwhelming for the Pango group (!), and they decided to raise the minimum number of sequences, at least for countries that can afford to sequence a lot of samples.  

"miscBA1BA2Post8393" is a much larger catch-all.  To illustrate, here is a screen grab from the Taxonium viewer (cov2tree.org) where I am viewing a local file with the complete UCSC/UShER tree (including both public and GISAID sequences, not public-only like the [public tree](https://hgdownload.gi.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/); I can share the complete tree with GISAID sequences with registered GISAID users, let me know if you are interested):
<img width="451" alt="image" src="https://user-images.githubusercontent.com/186983/164550759-e73bc99a-489f-4000-8aef-7ae506305d73.png">
Purple is miscBA1BA2Post8393; the dark red at the bottom is XE (truncated); the dark red somewhat up and to the right is XM; also shown are miscBA1BA2Pre10k, XH, XJ, proposed463, miscBA1BA2Post13k, proposed467, and then past XM, miscBA1BA2Post17k (light gray-green) and XK.

Notice how there is a lot of purple, with more specific branches & colorings interspersed.  There is a small red circle that shows the root node of the cluster from cov-lineages/pango-designation/issues/557 .  I think it's safe to say that most sequences in miscBA1BA2Post8393 are not directly related to the cluster from Canada -- they're just placed somewhat nearby because of overall similarity (and the impossibility in a tree structure of placing a recombinant adjacent to both of its parents, if its true parents are even included in the tree).

> In our case they were assigned XE by NextClade, and not XM.

Nextclade places your sequence on a tree to assign the lineage -- similar idea to UShER, but a separate tree maintained by @corneliusroemer and Ivan Aksamentov, and slightly different placement algorithm.  Undesignated recombinants are inherently tough for a tree-based approach to get right.  It's possible for the most similar lineage based on placement to change from one data release of the tree to the next, especially as new lineages and/or sequences are added -- and even more so between trees constructed by different methods like Nextclade vs. UShER.  In general there is a lot of room for improvement in how recombinants are analyzed and reported!

**Comment by ilevade** [comment]
Totally understandable, thank you so much for the quick reply. We will continue the monitoring and will let you know if the number of sequences from this cluster increase. Thanks again !
